ERROR - 2021-12-03 04:54:45 --> Table 'project_manager_db.ci_sessions' doesn't exist
CRITICAL - 2021-12-03 04:54:45 --> Call to a member function getRow() on bool
#0 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->read('c49c8762b001032...')
#1 /var/www/html/system/Session/Session.php(1001): session_start()
#2 /var/www/html/system/Session/Session.php(249): CodeIgniter\Session\Session->startSession()
#3 /var/www/html/system/Config/Services.php(736): CodeIgniter\Session\Session->start()
#4 /var/www/html/system/Config/BaseService.php(263): CodeIgniter\Config\Services::session(Object(Config\App), false)
#5 /var/www/html/system/Config/BaseService.php(192): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#6 /var/www/html/system/Config/Services.php(721): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#7 /var/www/html/system/Config/BaseService.php(263): CodeIgniter\Config\Services::session()
#8 /var/www/html/app/Models/Users_model.php(68): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#9 /var/www/html/app/Controllers/App_Controller.php(34): App\Models\Users_model->login_user_id()
#10 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#11 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#12 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#13 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#14 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-12-03 04:59:51 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 04:59:51 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:28:21 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:28:21 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:28:24 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:28:24 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:28:44 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:28:44 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:28:46 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:28:46 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:31:39 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:31:39 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:33:10 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:33:10 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:37:27 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:37:27 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:38:41 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:38:41 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-12-03 05:45:47 --> Error connecting to the database: Connection refused
CRITICAL - 2021-12-03 05:45:47 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
#0 /var/www/html/system/Database/BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/app/Models/Crud_model.php(26): CodeIgniter\Database\BaseConnection->query('SET sql_mode = ...')
#2 /var/www/html/app/Models/Settings_model.php(11): App\Models\Crud_model->__construct('settings')
#3 /var/www/html/system/Config/Factories.php(120): App\Models\Settings_model->__construct(NULL)
#4 /var/www/html/system/Common.php(904): CodeIgniter\Config\Factories::__callStatic('models', Array)
#5 /var/www/html/app/Controllers/App_Controller.php(31): model('App\\Models\\Sett...')
#6 /var/www/html/app/Controllers/Security_Controller.php(17): App\Controllers\App_Controller->__construct()
#7 /var/www/html/app/Controllers/Dashboard.php(11): App\Controllers\Security_Controller->__construct()
#8 /var/www/html/system/CodeIgniter.php(900): App\Controllers\Dashboard->__construct()
#9 /var/www/html/system/CodeIgniter.php(426): CodeIgniter\CodeIgniter->createController()
#10 /var/www/html/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/index.php(61): CodeIgniter\CodeIgniter->run()
#12 {main}
